develop-docs/application/feedback-architecture.mdx

```diff
@@ -186,12 +199,25 @@ filters, **skipping emails if:**
 
 ---
 
-## Feature flags for self-hosted
-To disable all UI features and/or ingestion, set these to false:
-* `organizations:user-feedback-ui`
-* `organizations:feedback-visible`
-* `organizations:user-feedback-ingest`
+## Self-hosted support
+
+This feature is available if you enable the following feature flags:
+
+- `organizations:user-feedback-ingest`
+- `organizations:user-feedback-ui`
+- `organizations:user-feedback-replay-clip`
+
+Auto-registered flags for issue platform (you won't be able to grep these in the sentry repo, but you need to enable them too):
+
+- `organizations:feedback-ingest`
+- `organizations:feedback-visible`
+- `organizations:feedback-post-process-group`
+
+User feedback will be available on a feature-complete version of Sentry by v24.8.0.
```

Comment:
The set up docs mention 24.4.2 for full functionality of UF, and 24.5.1 for attachments to be sent in the same envelope as feedback

Reply:
Uhh.. Wouldn't this caused misunderstandings? Since I believe on early days, the required feature flags for enabling User Feedback is quite different than what's currently being set?

Also I don't think `organizations:user-feedback-replay-clip` is available on 24.4.2.

Reply:
I think it's fine @aldy505 , if it helps I'll add a note in the feature flag section that the flags apply for 24.7.1+.